tools/google-cloud-support-slackbot/README.md

```diff
@@ -0,0 +1,153 @@
+	1. Click to expand **Management, security, disks, networking, sole tenancy**
+		1. Select the **Networking** tab
+		1. Under **Network interfaces**, click the network interface box
+			1. Set **Network** to the VPC where you have your firewall rules
+			1. Under **External IP**, select **Create IP address**. Choose whichever name and network service tier you prefer
+	1. Click **Create**
+
+### API Enablement and the API Key
+
+From **APIs & Services > Library** ...
+
+1. Search for and enable the **Cloud Logging API**
+2. Search for and enable the **Cloud Support API**
+
+From **APIs & Services > Credentials**
+
+1. Click **+Create** and select **API key**
+2. Copy your key and choose to **Restrict Key**
+	1. Under **Application restrictions**, you may select **IP addresses** to restrict usage the VM you created
+	1. Under **API restrictions**, select **Restrict Key** and from the **Select APIs** dropdown, click **Google Cloud Support API**
+
+## Setup Part 3 - Slack App
+
+Go to [Slack Apps](http://api.slack.com/apps) to do the following:
+
+1. Click **Create New App** and select **From scratch**. Name your app `Google Cloud Support Bot` and select your workspace
+2. Under **Settings > Basic Information**, scroll down to **Display Information** and upload the [google_cloud_support_buddy_big.png](google_cloud_support_buddy_big.png) or an icon of your choosing.
+3. Go to **Features > Slash Commands** and create the following command:
+	1. Command: `/google-cloud-support `
+	1. Request URL: `http://<your_vm_external_ip>/google-cloud-support`
+	1. Short description: `Track and manage your Google Cloud support cases in Slack. Use /google-cloud-support help for the list of commands`
+	1. Usage Hint: `[command] [parameter 1] [parameter 2] [parameter 3]`
+4. Go to **Features > OAuth & Permissions**. Scroll down to **Scopes** and add the **chat:write** scope. Add the **commands** scope if it isn't listed already listed
+5. At the top of the **Features > OAuth & Permissions** page, under **OAuth Tokens for Your Workspace**, click **Install to Workspace**. Copy the token. You may need Slack admin approval to install the app.
+
+## Setup Part 4 - Google Cloud Phase 2
+
+Return to [Google Cloud](https://cloud.google.com/console) and from **Compute Engine > VM instances**, perform the following:
+
+1. SSH into the VM that you created in part 2 of this setup guide
+2. Run the following commands:
+	1. `sudo apt-get update`
+	1. `sudo apt-get -y install subversion`
+	1. `sudo apt-get -y install python3-pip`
+	1. `sudo apt-get -y install nginx`
+	1. `cd /`
+	1. `sudo svn export https://github.com/GoogleCloudPlatform/professional-services/trunk/tools/google-cloud-support-slackbot`
+	1. `cd /google-cloud-support-slackbot`
+	1. Use sudo to open the `default` file with your editor of choice, and replace <STATIC_IP> with the external ip address of your VM. Then save and close the file
```

Comment:
Was this item supposed to be further on in the command list? It does not contain a command line command to be executed like the previous points.

Reply:
This is intentional, the commands for this aren't listed out as I assume most of the people who will be setting this up will have a basic knowledge of vim or nano.